client/www/pages/docs/presence-and-topics.md

```diff
@@ -136,12 +136,20 @@ const user = {
 };
 
 function App() {
-  const { user: myPresence, peers, publishPresence } = db.rooms.usePresence(room);
+  const {
+    user: myPresence,
+    peers,
+    publishPresence,
+  } = db.rooms.usePresence(
+    room,
+    // Publish your presence when you join the room
+    { initialData: { name: user.name } },
```

Comment:
nit: I _think_ we may need to also update `db.rooms.useSyncPresence`, so it passes the whatever it initially gets to `initialData` as well.

Reply:
Already done! https://github.com/instantdb/instant/pull/1176/files#diff-5350afe3652729d266c03240b7ebd1708380c05a31efc68c547968f819f31f90L208